package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "lerna run lint",
     "cs-check": "lerna run cs-check",
     "cs-format": "lerna run cs-format",
-    "build": "lerna run --stream build",
+    "build": "NODE_OPTIONS=--max-old-space-size=4096 lerna run --concurrency 1 --stream build",
```

Comment:
This isn't desirable, it's pretty slow as is. Are you running out of memory locally? If not, these options should only be used in CI.

We could also set the memory higher, linux VMs in GH actions have 7GB RAM and 2 cores.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
I'll revert this and try updating the `NODE_OPTIONS` in the `ci.yml`